bazel/repositories.bzl

```diff
@@ -116,6 +116,10 @@ def envoy_api_deps(skip_targets):
         name = "envoy_eds",
         actual = "@envoy_api//api:eds",
     )
+    native.bind(
```

Comment:
This seems like the wrong if statement to have this under? I will defer to @htuch for review on all of the bazel changes.